circle.yml

```diff
@@ -14,9 +14,10 @@ dependencies:
     - lsb_release -si -sr
```

Comment:
When the `lsb_release` got merged I think this was intended to be a `-d` argument.  Would you care to make that correction now?  I'd appreciate it.